Sources/KituraContracts/CodableQuery/Extensions.swift

```diff
@@ -200,6 +201,86 @@ extension String {
         return nil
     }
 
+    /**
+    Converts the given String to a [Date]? object using timeIntervalSince1970 in seconds.
+
+    - Returns: The [Date]? object. Some on success / nil on failure.
+    */
+    public func dateArray1970() -> [Date]? {
```

Comment:
I wonder if, rather than adding many more extensions to String, we could enhance (or overload) the existing `date()` function to handle each of these cases (by passing in a `DateDecodingStrategy`).